package.json

```diff
@@ -104,11 +104,14 @@
     "postinstall": "composer install",
     "prebuild": "npm install && composer install",
     "prepare": "is-ci || husky install",
-    "build": "npm run uglify && npm run makepot && npm run sass && rimraf build/* && npm run build:webpack && npm run build:release",
+    "build": "npm run uglify && npm run sass && rimraf build/* && npm run build:webpack && npm run build:i18n && npm run build:release",
+    "build:i18n": "npm run i18n:makepot && npm run i18n:merge",
```

Comment:
This task doesn't work if we run it directly.

```
[ERROR] Unable to load JS transpiling mappings from 'build/*.js.map' files. Make sure the JS assets compilation was successful.
```

It would be nice whether to make it work or specify how do we get the `*.js.map` files it needs.

Reply:
Added an example of compilation script to the error message:
`... Make sure the JS assets compilation was successful (e.g. npm run build:webpack).`